models/glm5.2_v20.md

```
@@ -66,24 +67,25 @@ fingerprint derived from the vLLM and SparkInfer commits.
 PUSH_IMAGE=1 ./build-gilded-gnosis-v20-final-cu132.sh
 ```
 
 The review for the build, embedded helper, calibrator, and their tests is
-[blackwell-llm-docker #5](https://github.com/local-inference-lab/blackwell-llm-docker/pull/5).
+[blackwell-llm-docker #6](https://github.com/local-inference-lab/blackwell-llm-docker/pull/6).
 
 The build deliberately excludes the separate weight-lifetime experiments in
 vLLM PR #154, vLLM PR #157, and SparkInfer PR #62. It also excludes the
 experimental sparse-CKV decode stack in vLLM PRs #159-#161 and SparkInfer PRs
-#64-#65.
+#64-#65. It also excludes the later `bounded_compat` commits on build PR #5;
```

markdownlint-cli2 (0.23.0)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)